src/sentry/static/sentry/app/views/organizationMonitors/monitorModel.jsx

```diff
@@ -0,0 +1,18 @@
+import FormModel from 'app/views/settings/components/forms/model';
+
+export default class MonitorModel extends FormModel {
+    getTransformedData() {
+      return Object.entries(this.fields.toJSON()).reduce((data, [k, v]) => {
+        if (k.indexOf('config.') === 0) {
+          if (!data.config) data.config = {};
+          data.config[k.substr(7)] = v;
+        }
+        return data;
+      }, {});
+    }
+
+    getTransformedValue(id) {
```

Comment:
Hmmm I don't think you need this, this gets called by `FormModel.getTransformedData` which you've overridden

Reply:
hrm might have done this before I realized the default getTransformedData was problematic for me (was using getData originally)

Reply:
it gets called in saveFieldRequest as well

Reply:
that's if you use save on blur functionality

Reply:
Ill keep it for correctness then